dentapp/src/components/Category.js

```diff
@@ -0,0 +1,58 @@
+            <circle cx="59.3264" cy="36" r="25" fill={bgColor} fill-opacity="0.6" />
+        </svg>)
+}
+
+class Category extends Component {
+
+    state = {};
+
+    render() {
+        const { bgColor, title, icon, onClick } = this.props;
+        return (
+            <>
+                <div onClick={onClick()} className="text-center" style={{width: "102px", height:"130px"}}>
```

Comment:
onClick={onClick} instead of onClick={onClick()}, otherwise the function will execute inmediately when the component is mounted before doing click